### PR TITLE
Update with Data Umbrella events: Nov & Dec 2024

### PR DESCRIPTION
**Pull request by reshamas**
I made some minor updates:  
- add the November Polars/Narwhals back in
- fix the event description for Nov event
- update the event description for Dec event (Git)

I didn't realize we could keep prior events, so moving forward, let's do that. We can put the most recent event at the top.

For the past event, I am adding in the YouTube video link. Let's see if that works. 

cc: [mmcelaney](https://github.com/mmcelaney)

### CODE REVIEW

**Comment by reshamas** [comment]
@stefanv Would it be possible for us to add the YouTube link?
Or is it not possible in the template?

**Comment by bsipocz** [comment]
I see the TY link for the narwals entry showing up in the preview.

**Review by bsipocz — APPROVED** [review]

File: calendars/data-umbrella.yaml [hunk 1]
```diff
@@ -2,12 +2,31 @@ name: Data Umbrella Events Calendar
...
-      that is happening in the NumPy community. Everyone is welcome
-      to attend and contribute to a conversation.
+      Join us for an introductory session on Git. Discover some
+      fundamental concepts, explore its practical applications, and
+      learn how to efficiently use both the command line and the Visual
+      Studio Code graphical user interface (GUI).
 
       Meetup: https://www.meetup.com/data-umbrella/events/304521945/
       Webinarlink: https://www.bigmarker.com/neo4j/Data-Umbrella-Webinar
+      Video: Will be posted on YT
```
Comment (reshamas) [inline calendars/data-umbrella.yaml:12]: Video: https://www.youtube.com/@DataUmbrella

**Comment by reshamas** [comment]
Thanks @bsipocz 
I wonder if it will take that main YT link for our upcoming events. 
I wasn't able to make the commit on GitHub.

File: calendars/data-umbrella.yaml (hunk 1 above)
Comment (stefanv) [inline calendars/data-umbrella.yaml:12]:
```suggestion
      Video: https://www.youtube.com/@DataUmbrella
```

**Comment by reshamas** [comment]
I think this is ready to be merged?

**Review by jarrodmillman — APPROVED** [review]

**Pull request merged**